Hooks/BuffHook.cs

```diff
@@ -76,6 +76,9 @@ private static void SpawnPrefix(ModifyUnitStatBuffSystem_Spawn __instance)
         
         foreach (var entity in entities)
         {
+            //not sure if this goes here
+            if (Plugin.WeaponMasterySystemActive) WeaponMasterySystem.ApplyWeaponBuffs(entity, entityManager);
```

Comment:
Don't do this here. It should be done as a part of the `ApplyBuffs` so that we do not clash with other mods.